Tasks/XamarinTestCloud/xamarintestcloud.ts

```diff
@@ -229,7 +229,10 @@ var submitToTestCloud = function (index) {
     // For an iOS .ipa app, look for an accompanying dSYM file
     if (dsym && path.extname(appFiles[index]) == '.ipa') {
         // Find dSYM files matching the specified pattern
-        var alldsymFiles = tl.find(path.dirname(appFiles[index]));
+        // Check in one folder up since IPAs are now generated under a timestamped folder
+        var ipaFolder = path.dirname(path.dirname(appFiles[index]));
```

Comment:
@jpricketMSFT: To mock path.dirname, we have to create a dirname method in the task-lib. I will check if the new L0 framework allows a simpler way to mock this.